lib/Core/ExecutorUtil.cpp

```diff
@@ -59,6 +59,12 @@ namespace klee {
       } else if (isa<ConstantPointerNull>(c)) {
         return Expr::createPointer(0);
       } else if (isa<UndefValue>(c) || isa<ConstantAggregateZero>(c)) {
+        if (getWidthForLLVMType(c->getType()) == 0){
```

Comment:
@futile Could you also clang-format the patch?  In particular spacing before `{` is inconsistent with the surrounding code.